plugin-enclave-gradle/build.gradle

```diff
@@ -1,16 +1,10 @@
+//  Note: the settings specifying the mapping of this plugin
+//    to its implementation class are in the root build.gradle
+
 plugins {
-    id 'java-gradle-plugin'
     id 'org.jetbrains.kotlin.jvm'
 }
 
-gradlePlugin {
-    plugins {
-        enclavePlugin {
-            id = 'com.r3.conclave.enclave'
-            implementationClass = 'com.r3.conclave.plugin.enclave.gradle.GradleEnclavePlugin'
-        }
-    }
-}
 
```

Comment:
I have moved this to the root `build.gradle` as `java-gradle-plugin` works with `maven-publish` plugin during publication and this issues if we do not coordinate the two together in the same place (i.e. for a developer it is a pain to discover that a change in `build.gradle` is also affected by the `java-gradle-plugin` here. This is in the context of the evaluation of the project.

Reply:
Can this not be resolved by just commenting in the root build.gradle. See my comment above why I'd prefer this config to stay here.

Reply:
Moved back